apps/web/app/(ee)/partners.dub.co/(dashboard)/programs/[programSlug]/(enrolled)/bounties/page-client.tsx

```
@@ -0,0 +1,68 @@
+                <BountyCardSkeleton key={index} />
+              ))}
+        </div>
+      ) : (
+        <AnimatedEmptyState
+          title="No bounties to collect"
+          description={
+            <>
+              This program isn't offering any bounties at the moment.{" "}
+              <a
+                href="https://dub.co/help/article/bounties"
+                target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)